tools/c7n_oci/tests/cassettes/TestInstance.test_instance_metrics_by_chunks.yml

```diff
@@ -0,0 +1,1823 @@
+interactions:
+- request:
+    body: null
+    headers: {}
+    method: GET
+    uri: https://identity.us-ashburn-1.oci.oraclecloud.com/20160918/users/ocid1.user.oc1..<unique_ID>
+  response:
+    body:
+      data:
```

Comment:
it feels like have lots of extraneous recorded data here re instances not relevant to the test.

Reply:
done